storage/memory/registers.go

```diff
@@ -0,0 +1,93 @@
+package memory
```

Comment:
I guess we won't need this implementation once you rebase with `master`

Reply:
Yes 100%